lib/request.js

```diff
@@ -0,0 +1,81 @@
+`use strict`
+
+const http = require('http');
+const https = require('https');
+const url = require('url');
+const debug = require("debug")("universal-analytics");
+
+function getProtocol(path) {
+  return url.parse(path).protocol === "http:" ? http : https;
```

Comment:
Would it help to inline the `require` so only the module that is used is getting loaded?